picard/ui/searchdialog.py

```diff
@@ -0,0 +1,301 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+from PyQt4 import QtGui, QtCore
+from operator import itemgetter
+from functools import partial
+from picard import config
+from picard.file import File
+from picard.ui import PicardDialog
+from picard.ui.util import StandardButton, ButtonLineEdit
+from picard.util import format_time, icontheme
+from picard.mbxml import artist_credit_from_node
+
+
+class TracksTable(QtGui.QTableWidget):
+
+    def __init__(self, parent=None):
+        QtGui.QTableWidget.__init__(self, 0, 7)
+        self.setHorizontalHeaderLabels([_("Name"), _("Length"),
```

Comment:
Since those are translatable, i would put one per line. It is simpler to review/diff/po update any change.